### DISCUSSION TITLE
Issue with Find and Side Menu After Updating ngx-extended-pdf-viewer from 21.4.1 to 21.4.6

### DISCUSSION THREAD

**Discussion by aathi271991** [stephanrauh/ngx-extended-pdf-viewer#2674] [General]
Hi everyone,

I recently updated ngx-extended-pdf-viewer from version 21.4.1 to 21.4.6. After the update, the Find tool and Side Menu buttons are visible, but their click actions are no longer working.

Everything was working fine before the update. Has anyone else experienced this issue? Could it be related to a configuration change or a bug in the new version?
![Screenshot 2024-11-30 000838](https://github.com/user-attachments/assets/15c6a1ca-950d-462d-8f24-860a079127bf)

**Comment by stephanrauh** [comment]
Either go back to version 21.4.5 or update to version 22.

When publishing version 21.4.6, I made a mistake. It was meant to be the first alpha of version 22, but I forgot to update the version number and only noticed after publishing. Unfortunately, npm doesn't allow me to delete a version.

At the moment, version 22 is still an alpha version, but that's only because I didn't find enough spare time to test everything. I believe the quality of version 22 should be OK by now.

**Reply by stephanrauh** [reply]
Version 22.0.0 has landed on npm. 

Enjoy!
Stephan

**Comment by aathi271991** [comment]
Thank you very much

Thanks & Regards,
Srimurugan
